### PR TITLE
Rename scene to granule in output metadata file

### PR DESCRIPTION
**Pull request by forrestfwilliams**
TODO:
- [x] Change naming of scenes -> granules
- [x] Remove parallel baseline
- [x] Check calculation of perpendicular baseline (should not be negative)

### CODE REVIEW

**Comment by forrestfwilliams** [comment]
@jtherrmann or @AndrewPlayer3 ready for review!

File: src/hyp3_isce2/insar_tops_burst.py [hunk 1]
```diff
@@ -172,8 +173,8 @@ def make_parameter_file(
     slant_range_time = float(ref_annotation_xml.find('.//slantRangeTime').text)
     range_sampling_rate = float(ref_annotation_xml.find('.//rangeSamplingRate').text)
     number_samples = int(ref_annotation_xml.find('.//swathTiming/samplesPerBurst').text)
-    baseline_par = topsProc_xml.find(f'.//IW-{swath_number}_Bpar_at_midrange_for_first_common_burst').text
-    baseline_perp = topsProc_xml.find(f'.//IW-{swath_number}_Bperp_at_midrange_for_first_common_burst').text
+    # FIXME: Not sure why baseline is negative in some cases. I believe it is just a convention
+    baseline_perp = np.abs(topsProc_xml.find(f'.//IW-{swath_number}_Bperp_at_midrange_for_first_common_burst').text)
```
Comment (jtherrmann) [inline src/hyp3_isce2/insar_tops_burst.py:177]: Should we create an issue for this?

**Review by jtherrmann — DISMISSED** [review]

File: src/hyp3_isce2/insar_tops_burst.py (hunk 1 above)
Comment (asjohnston-asf) [reply src/hyp3_isce2/insar_tops_burst.py:177]: @cirrusasf reports our hyp3-gamma products also sometimes report negative baseline values. I see that's the case for 2/5 of the interferograms I happen to have in my downloads folder:
```
$ grep -R Baseline: S1*/*.txt
S1AA_20220927T164913_20221102T164913_VVR036_INT80_G_ueF_306C.txt:Baseline: 30.9803
S1AA_20230128T153358_20230209T153357_VVR012_INT40_G_weF_E42F.txt:Baseline: 176.6564
S1AA_20230128T153424_20230209T153423_VVR012_INT40_G_weF_216F.txt:Baseline: 174.217
S1AA_20230129T033427_20230210T033426_VVR012_INT80_G_weF_534B.txt:Baseline: -105.159
S1AA_20230129T033452_20230210T033451_VVR012_INT80_G_weF_85A3.txt:Baseline: -107.6773
```

Comment (asjohnston-asf) [reply src/hyp3_isce2/insar_tops_burst.py:177]: For example, this job: https://hyp3-api.asf.alaska.edu/jobs/628a66ed-aa96-4d03-8bfd-fb9b9a7fd671
```
S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8
S1A_IW_SLC__1SSV_20150504T120217_20150504T120229_005771_00769E_EF9A
```
gives a baseline of `-77.6382`

Comment (jtherrmann) [reply src/hyp3_isce2/insar_tops_burst.py:177]: And this command:

```bash
python -m hyp3_isce2 --reference-scene S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8 --secondary-scene S1A_IW_SLC__1SSV_20150504T120217_20150504T120229_005771_00769E_EF9A --swath-number 1 --reference-burst-number 1 --secondary-burst-number 1
```

gives a perpendicular baseline of `-81.53440934262281`.

Comment (jtherrmann) [inline src/hyp3_isce2/insar_tops_burst.py:177]:
```suggestion
    baseline_perp = topsProc_xml.find(f'.//IW-{swath_number}_Bperp_at_midrange_for_first_common_burst').text
```

Comment (jtherrmann) [reply src/hyp3_isce2/insar_tops_burst.py]: Moving further discussion to https://github.com/ASFHyP3/hyp3-isce2/issues/62

**Review by jtherrmann — APPROVED** [review]

**Review by jtherrmann — APPROVED** [review]

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**